starfish/imagestack/imagestack.py

```diff
@@ -290,6 +291,37 @@ def xarray(self) -> xr.DataArray:
         """Retrieves the image data as an xarray.DataArray"""
         return self._data
 
+    def _convert_to_indexers_dict(self, pos_kwargs):
```

Comment:
It would be helpful to add a docblock to this method.

Reply:
Also, I don't understand what the value of this as a method is.  It is only used once, and both the caller and this method are pretty simple methods...

Reply:
It does three things 1.) creates an empty slice(None, None) entry for every dimensions in indexers so that we don't have to add a bunch of "If Indices.X in indexers" statements down the line. 2.) Converts the dict keys from Indices to their string values which is needed to call xarray.sel(indexers). 3.) converts tuple range values to slice objects. Do you mean I should just add all this code to the sel method?

Reply:
Also if we ever did want to add indexing with kwargs or another ux for indexing it would be easier to have this separate to do all the index parsing.

Reply:
So you should put the explanation in the docblock.  And possibly inline this in the `sel` method.

Reply:
> Also if we ever did want to add indexing with kwargs or another ux for indexing it would be easier to have this separate to do all the index parsing.

understood.  then just a request for docblock.